src/components/Pill/index.js

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable import/prefer-default-export */
+
+export { default as PillButton } from './PillButton';
+// export { default as PillDropdown } from './PillDropdown';
+// export {default as Pill } from './Pill'
```

Comment:
shall we remove these commented exports?

Reply:
I'd rather keep them as placeholder for the next tickets that needs implementing so that the relation between this and the upcoming parts is clear.